.buildkite/pipeline.yml

```diff
@@ -168,3 +168,30 @@ steps:
     plugins:
       - junit-annotate#v2.4.1:
           artifacts: build/TEST-go-integration*.xml
+
+  # Branch steps
+  - group: "K8s tests"
+    key: "k8s-tests"
+    steps:
+      - label: "K8s tests: {{matrix.k8s_version}}"
+        branches: "main 7.* 8.* v7.* v8.* buildkite_k8s_tests"
+        env:
+          K8S_VERSION: "v{{matrix.k8s_version}}"
+          KIND_VERSION: "v0.20.0"
+        command: ".buildkite/scripts/steps/k8s-tests.sh"
+        artifact_paths:
+          - "build/TEST-**"
+          - "build/diagnostics/*"
+          - "coverage.out"
+        agents:
+          provider: "gcp"
+          image: "family/core-ubuntu-2204"
+        matrix:
+          setup:
+            k8s_version:
+              - "1.28.0"
```

Comment:
I see you have aligned  with this https://github.com/elastic/elastic-agent/pull/3676?
cc @constanca-m to see any diff

Reply:
That's right. I picked the versions according to this PR